sql/src/main/java/io/crate/executor/transport/TransportFetchNodeAction.java

```diff
@@ -89,7 +89,7 @@ public void execute(
             final NodeFetchRequest request,
             ActionListener<NodeFetchResponse> listener) {
         transports.executeLocalOrWithTransport(this, targetNode, request, listener,
-                new DefaultTransportResponseHandler<NodeFetchResponse>(listener, executorName()) {
+                new DefaultTransportResponseHandler<NodeFetchResponse>(listener, ThreadPool.Names.SUGGEST) {
```

Comment:
minor: imho using a class variable would highlight the used threadpool a bit more